scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -2,72 +2,72 @@ package org.scalafmt.util
 
 import java.io.File
 import java.nio.charset.StandardCharsets
-import java.nio.file.{Files, Paths}
-import scala.util._
-import org.scalactic.source.Position
+import java.nio.file.{Files, Path, Paths}
+
+import munit.{Assertions, FunSuite}
 import org.scalafmt.util.DeleteTree.deleteTree
-import org.scalatest._
-import org.scalatest.funsuite
-import org.scalatest.Assertions
 
-class GitOpsTest extends funsuite.FixtureAnyFunSuite {
+import scala.util._
+import munit.Location
+
+class GitOpsTest extends FunSuite {
 
   import GitOpsTest._
 
   val root = AbsoluteFile.userDir
   val dirName = "gitTestDir"
 
-  override type FixtureParam = GitOpsImpl
-
   // DESNOTE(2017-08-16, pjrt): Create a temporary git directory for each
   // test.
-  override def withFixture(test: OneArgTest): Outcome = {
-    val f = Files.createTempDirectory(dirName)
-    val absFile = AbsoluteFile.fromPath(f.toString).get
-    val ops = new GitOpsImpl(absFile)
-    init(ops)
+  private implicit var gitOps: GitOpsImpl = _
```

Comment:
why not call this `ops`, just like before, and vastly reduce the size of this patch?

Reply:
Sure, it really came from the revert.

Reply:
Done!